client/src/components/SupportTools/SupportTools.tsx

```diff
@@ -484,28 +469,24 @@ const Jurisdiction = ({ jurisdictionId }: { jurisdictionId: string }) => {
             <p>The jurisdiction hasn&apos;t created audit boards yet.</p>
           ) : (
             <>
-              <Button intent="danger" onClick={onClickClearAuditBoards}>
+              <Button
+                intent="danger"
+                onClick={onClickClearAuditBoards}
+                style={{ marginBottom: '10px' }}
+              >
                 Clear audit boards
               </Button>
-              <Table striped>
-                <tbody>
-                  {auditBoards.map(auditBoard => (
-                    <tr key={auditBoard.id}>
-                      <td>{auditBoard.name}</td>
-                      {election.online && (
-                        <td>
-                          <Button
-                            onClick={() => onClickReopenAuditBoard(auditBoard)}
-                            disabled={!auditBoard.signedOffAt}
-                          >
-                            Reopen
-                          </Button>
-                        </td>
-                      )}
-                    </tr>
-                  ))}
-                </tbody>
-              </Table>
+              <AuditBoardsTable
+                areAuditBoardsOnline={election.online}
+                auditBoards={auditBoards}
+                reopenAuditBoard={async auditBoard => {
+                  await reopenAuditBoard.mutateAsync({
+                    auditBoardId: auditBoard.id,
+                    jurisdictionId,
+                  })
+                  toast.success(`Reopened ${auditBoard.name}`)
+                }}
+              />
```

Comment:
What if we just remove this feature for support users completely? In general, the approach I've been taking is to only have support features for things that users can't do themselves. Support users can always log in as a user if they need to do something on the user's behalf. Leaves us with less code, which is easier to maintain.

Reply:
Great call! Will remove the "Reopen" buttons and the corresponding support user API endpoints. I'll leave the list of audit boards since that's still potentially useful context for the support-user specific "Clear audit boards" button